### ISSUE TITLE
Output variable description

### ISSUE THREAD

**Issue by panditk17** [EDmodel/ED2#204]
Hi,
I could not understand some of the output variables (eg, A_CO2, CCWD_FLUX, CBUDGET_NEP, MOIST_F, NCWD_FLUX, WAI_CO etc). Is there anywhere I could look into to know more on the proper names and units of these variables? Thanks.

**Comment by manfredo89** [comment]
For most (though not all) output variables you have a description in the metadata. This is an option that you have to switch in the ED2IN. So if you set
NL%ATTACH_METADATA = 1
in your ED2IN you will then have a description of your variable.
Which kind of output are you looking at? Because beside WAI_CO I don't see the variables in the Q (monthly means) files.

**Comment by panditk17** [comment]
Thank you very much. I have NL%ATTACH_METADATA=1 in ED2IN, but still cant figure out how to extract the description of the variables. I tried to open .h5 outputs to look into variable description but could not locate them. Thanks. I was looking into S (daily ouputs) for my variables.

**Comment by ashehad** [comment]
Maybe you want to take a look at here...https://raw.githubusercontent.com/EDmodel/ED2/master/ED/src/memory/ed_state_vars.F90

**Comment by xiangtao-princeton** [comment]
@panditk17 I agree with ashehad that currently the best way is to check the monster file ed_state_vars.F90 and search the variable names. Please close the issue if it is resolved.
